src/ExtensionsSample/Samples/NotificationHubsSamples.cs

```diff
@@ -0,0 +1,57 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+using System.Collections.Generic;
+using Microsoft.Azure.NotificationHubs;
+using Microsoft.Azure.WebJobs;
+using Microsoft.Azure.WebJobs.Extensions.NotificationHubs;
+
+namespace ExtensionsSample
+{
+    // To use the NotificationHubsSample:
+    // 1. Create a new Mobile App
+    // 2. Create and configure NotificationHub
+    // 3. Add the NotificationHubs connection string to a 'AzureWebJobsNotificationHubConnectionString' App Setting in app.config    
+    // 4. Add the NotificationHubs Hub name to a 'AzureWebJobsNotificationHubName' App Setting in app.config    
+    // 5. Use MobileApps client SDK to register template with NotificationHubs
+    public static class NotificationHubsSamples
+    {
+        // The binding sends push notification to any clients registered with the template
+        //   method successfully exits.
+        public static void SendNotificationOnTimerTrigger(
+            [TimerTrigger("*/15 * * * * *")] TimerInfo timerInfo,
+            [NotificationHubs] out Notification notification)
+        {
+            notification = GetTemplateNotification("bar");
+        }
+
+        // The binding sends multiple push notification to any clients registered with the template
+        //   method successfully exits.
+        public static void SendNotificationsOnTimerTrigger(
+            [TimerTrigger("*/30 * * * * *")] TimerInfo timerInfo,
+            [NotificationHubs] out Notification[] notifications)
+        {
+            notifications = new TemplateNotification[]
+                {
+                    GetTemplateNotification("bar1"),
```

Comment:
nit: for all these test strings, rather than foo/bar, can you use "Message 1"/"Message 2" or something like that?